doc/theory/glossary.tex

```diff
@@ -396,54 +387,66 @@
 	
 	The name is inspired from the case when the persistence diagram comes from persistent homology.
 	
-	\subsection*{Distances, inner products and kernels} \label{metric_inner_product_and_kernel}
+	\subsection*{Metric space} \label{metric space}	
+	A set $X$ with a function	
+	\begin{equation*}	
+	d : X \times X \to \mathbb R	
+	\end{equation*}	
+	is said to be a \textit{metric space} if the values of $d$ are all non-negative and for all $x,y,z \in X$ ans $a \in \mathbb R$	
+	\begin{equation*}	
+	d(x,y) = 0\ \Leftrightarrow\ x = y	
+	\end{equation*}	
+	\begin{equation*}	
+	d(x,y) = d(y,x)	
+	\end{equation*}	
+	\begin{equation*}	
+	d(x,z) \leq d(x,y) + d(y, z).	
+	\end{equation*}	
+	In this case the $d$ is referred to as the \textit{metric} or the \textit{distance function}.	
 	
-	A set $X$ with a function
-	\begin{equation*}
-	d : X \times X \to \mathbb R
-	\end{equation*}
-	is called a \textit{metric space} if the values of $d$ are all non-negative and for all $x,y,z \in X$
-	\begin{equation*}
-	d(x,y) = 0\ \Leftrightarrow\ x = y
-	\end{equation*}
-	\begin{equation*}
-	d(x,y) = d(y,x)
-	\end{equation*}
-	\begin{equation*}
-	d(x,z) \leq d(x,y) + d(y, z).
-	\end{equation*}
-	In this case the $d$ is referred to as the \textit{metric} or the \textit{distance function}.
+	\subsection*{Inner product and norm} \label{inner_product_and_norm}	
 	
 	A vector space $V$ together with a function
-	\begin{equation*}
-	\langle -, - \rangle : V \times V \to \mathbb R
-	\end{equation*}
-	is called and \textit{inner product space} if for all $u,v,w \in V$
-	\begin{equation*}
-	u \neq 0\ \Rightarrow\ \langle u, u \rangle > 0 
+	\begin{equation*}	
+	\langle -, - \rangle : V \times V \to \mathbb R	
 	\end{equation*}
-	\begin{equation*}
-	\langle u, v\rangle = \langle v, u\rangle
-	\end{equation*}
-	\begin{equation*}
-	\langle au+v, w \rangle = a\langle u, w \rangle + \langle v, w \rangle.
-	\end{equation*}
-	In this case the function $\langle -, - \rangle$ is referred to as the \textit{inner product} and the function given by
-	\begin{equation*}
-	||u|| = \sqrt{\langle u, u \rangle}
+	is said to be an \textit{inner product space} if for all $u,v,w \in V$
```

Comment:
Or, `a\in\mathbb R` was meant to be here?

Reply:
Yep, will get it fixed.